weldx/asdf/util.py

```diff
@@ -18,6 +21,21 @@
     "notebook_fileprinter",
 ]
 
+
+def get_schema_path(schema: str) -> Path:
```

Comment:
could a schema argument include a "path" to get a subtree?
e.g. "single_proc.geometry" or "single_proc.process.wire_feed"

Reply:
eventually this should be a separate function

Reply:
yeah this can be beefed up later for sure